static/coffee/modals.coffee

```diff
@@ -90,13 +90,25 @@ modals.controller 'NewCaseModalController', ['$scope', '$uibModalInstance', 'sum
 #=====================================================================
 # Assign to partner modal
 #=====================================================================
-modals.controller 'AssignModalController', ['$scope', '$uibModalInstance', 'title', 'prompt', 'partners', ($scope, $uibModalInstance, title, prompt, partners) ->
+modals.controller 'AssignModalController', ['$scope', '$uibModalInstance', 'title', 'prompt', 'partners', 'users', 'UserService', ($scope, $uibModalInstance, title, prompt, partners, users, UserService) ->
   $scope.title = title
   $scope.prompt = prompt
   $scope.partners = partners
-  $scope.fields = { assignee: partners[0] }
 
-  $scope.ok = () -> $uibModalInstance.close($scope.fields.assignee)
+  user_list_from_users = (users) ->
```

Comment:
Resolved in https://github.com/praekelt/casepro/pull/69/commits/12ebb14955f6d904afb61ba3ef0cda7521880980